src/Mod/Path/PathScripts/PathPost.py

```diff
@@ -236,7 +236,7 @@ def Activated(self):
             elif hasattr(sel, "Path"):
                 try:
                     job = PathUtils.findParentJob(sel)
-                except Exception: # pylint: disable=broad-except
+                except Exception:
```

Comment:
Just curious, is there a reason why you remove the pylint suppressions added by @mlampert in https://github.com/FreeCAD/FreeCAD/commit/f16703e9496e3b651676c126775493ce1cb1f9e1 ?

Reply:
I might be misrembering a conversation with mlampert.  I thought they were deprecated now

Reply:
those were added when we didn't have any other linter for python - I think we should remove them as we go, we're not using pylint